### ISSUE TITLE
parameter initialization discrepancy among flowClust APIs

### ISSUE THREAD

**Issue by mikejiang** [RGLab/flowClust#15]
There are two ways of using `flowClust`, either through `flowClust` directly
```r
flowClust(fr, params, K = K)
```
Or through `tmixFilter`
```r
 tmix_filter <- tmixFilter(params, K = K)
 filter(fr, tmix_filter)
```
You would expect they return the same clustering results but as @gfinak and I just found out they actually don't.

It turns out that one of the parameters is set to different values through two APIs, i.e.
`randomStart` is set to `0` for  [flowClust](https://github.com/RGLab/flowClust/blob/trunk/R/flowClust.R#L1) but  `10` for [tmixFilter](https://github.com/RGLab/flowClust/blob/trunk/R/SetClasses.R#L69-L73), which leads to the entirely different behaviors.

To avoid the future confusion, I think we may set it to `10` for both since another internal API [.flowClustK](https://github.com/RGLab/flowClust/blob/trunk/R/flowClust.R#L159) also uses `10`.

It would be also helpful to pass most of parameters from `flowClust` to `.flowClustK` through `...` to avoid the potential discrepancy.

**Comment by gfinak** [comment]
No, set it to 0, and use the kmeans initialization. The random initialization is not converging properly and needs to be debugged.

**Comment by mikejiang** [comment]
Sure

**Comment by gfinak** [comment]
I've tracked down the issue. The `tol` parameter needs to be decreased. Default is 1e-5, in cases where it doesn't converge after random initialization, it should be lowered, one or two orders of magnitude should be sufficient.
